src/components/Header/SideNav/Dropdown.js

```diff
@@ -59,8 +60,17 @@ export default class Dropdown extends PureComponent {
   }
 
   render() {
-    const { items, children } = this.props
-    const { isExpanded } = this.state
+    const { items, children, path } = this.props
+    let { isExpanded } = this.state
+
+    const isServicePage = path.search(servicesRegExp) > -1
```

Comment:
I think this is already written somewhere else. In utils?

Reply:
Good call. I added that 2 places for a previous ticket (TopNav components) but that wasn't part of a utils file. Now that it's used here as well it makes sense to move it to a utils file at the /Header level

Reply:
I moved it 👍